src/index.d.ts

```diff
@@ -469,4 +469,13 @@ export interface IEvents {
     type: string
     payload: any
   }
+
+  videoEvent: {
+    componentId: string
+    sceneId: string
+    videoClipId: string
+    videoStatus: number
```

Comment:
- Please add comments in `videoStatus` about possible values.
- I think `sceneId` is not necessary to be declared here.
- I'd be more specific in the name `length` -> `totalVideoLength` to prevent confusions

Reply:
1) Got it
2) Right, I tought about videos from other scenes, but there wouldn't make sense
3) Ok. Should I add comment that it can be -1 (for streamings or no information)?